proposals/lip-0067.md

```diff
@@ -73,8 +72,9 @@ keystoreSchema = {
     },
     "ID": {
         "type": "string",
-        "format": "uuid"
-    }
+        "format": "id"
```

Comment:
What is the difference between "uuid" and "id" here?

Reply:
Updated with "type" field only and removed "format" field.